test/Orleans.Transactions.Tests/Runners/GoldenPathTransactionTestRunner.cs

```diff
@@ -80,10 +80,44 @@ protected GoldenPathTransactionTestRunner(IGrainFactory grainFactory, ITestOutpu
             foreach (var grain in grains)
             {
                 int actual = await grain.Get();
+                if (expected != actual) this.output.WriteLine($"{grain} - failed");
                 Assert.Equal(expected, actual);
             }
         }
 
+        [SkippableTheory]
+        [InlineData(TransactionTestConstants.SingleStateTransactionalGrain)]
+        public virtual async Task RepeatGrainReadWriteTransaction(string transactionTestGrainClassName)
+        {
+            const int repeat = 10;
```

Comment:
maybe repeat 20? I know you reproduced this bug by just repeat 10 times, but this kind of behavior is hard to control, especially this will be running in CI, instead of your local machine. Running it 20 should be safer, more assuring.

Reply:
10 has been sufficient and is still fast test, even against azure.  We can push higher if/when needed.